editoast/src/views/train_schedule.rs

```diff
@@ -1105,7 +1107,7 @@ mod tests {
         )
     }
 
-    fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
+    pub(in crate::views) fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
```

Comment:
Instead of changing the visibility, how about moving this function inside `src/views/mod.rs` and exposing it only for tests. Because if it's a function that is supposed to be used by 2 different submodules of `views`, then it's now a utility of `views`, not anymore a simple function of `view::train_schedule`.

```suggestion
    #[cfg(test)]
    pub fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
```